rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.88"
+targets = ["x86_64-unknown-linux-musl", "aarch64-unknown-linux-musl"]
```

Comment:
What is the semantics of the `rust-toolchain` file? Will it interfere with normal users building `nickel` locally without Nix? Also, I'm a bit worried we have yet a new config file to keep in sync.

Reply:
Ok, I moved the toolchain selection into the release artifacts job and checked that it still runs.

Reply:
Good point, we could read this file from Nix. There's still the risk that what's in rust_toolchain.toml is not available in Nixpkgs, but I suppose that's something we can deal with, given that the flake inputs are locked. In fact I think I would prefer this workflow as well in the future, rather than having a random flake.lock input update bumping the Rust version in an unforeseeable way; we would have more control on updating the toolchain.